integration-tests/__tests__/snapshot_resolver.test.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @flow
+ */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const runJest = require('../runJest');
+
+const snapshotDir = path.resolve(
+  __dirname,
+  path.join('..', 'snapshot-resolver', '__snapshots__'),
```

Comment:
`path.resolve` normalizes, so just `path.resolve(__dirname, '../snapshot-resolver/__snapshots__');` should be enough